packages/react-storage/src/components/StorageBrowser/displayText/en.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Temporarily just export strings until displayText strategy is finalized
+ */
+export const displayText = {
+  uploadDestination: 'Destination',
```

Comment:
Tangential to the actual PR: can genericize the key here, "destination" will be used for other actions (e.g. `copy`)

Reply:
I'll change it to actionDestination so it's not overly broad